bin/startup_imgsrv

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Usage: startup_imgsrv
+
+exec >> /tmp/imgsrv.out 2>> /tmp/imgsrv.err
+
+NPROC=4
+SDRDATAROOT=$SDRDATAROOT
+SDRROOT=$SDRROOT
+HT_DEV=
```

Comment:
No-op / duplicative of env vars declared outside?

Reply:
Cheap copy of the nebula template :-)

Reply:
Updated.